src/lib/serviceRoutinesV2/postEntities.cpp

```diff
@@ -34,6 +34,12 @@
 #include "serviceRoutines/postUpdateContext.h"
 
 
+static bool legalEntityLength(Entity* eP, const std::string &servicePath)
+{
+  const int constant = 10;
```

Comment:
#define (the one that we typically use in CB) has the advantage of pre-processor text replacement (i.e. faster code... although maybe marginally faster) while static const has the advantage of allowing semantics in the constant (so it is treated as `int` in every place were it is used). 

Both are ok, not an important issue at this point (use the one you prefer).